cmake/CheckTrezor.cmake

```
@@ -75,19 +75,25 @@ if (USE_DEVICE_TREZOR)
     # Protobuf is required to build protobuf messages for Trezor
     include(FindProtobuf OPTIONAL)
 
-    FIND_PACKAGE(Protobuf CONFIG)
+    # PkgConfig works better with new Protobuf
+    find_package(PkgConfig REQUIRED)
```

```suggestion
    find_package(PkgConfig QUIET)
```

at least that's what we use in `cmake/FindSodium.cmake`, is required here necessarsy?